test/fixtures/lightscript/comprehension/object-basic/actual.js

```diff
@@ -0,0 +1 @@
+{for i=0; i<10; i++: (i,i)}
```

Comment:
sorry, should have clarified that one of the reasons is that they just generate much larger AST's than simpler for-loops, making them harder to review